### PR TITLE
DCP-3013 - adding unit test cases

### PR DESCRIPTION
**Pull request by swatisuman1990**

### CODE REVIEW

File: command-line-downloader/src/test/java/uk/ac/ebi/ena/backend/EnaPortalServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        verify(restTemplate, times(1)).postForObject(Mockito.any(URI.class), Mockito.any(HttpEntity.class), Mockito.eq(String.class));
+    }
+
+    private EnaPortalResponse[] getPortalResponses() {
+        List<EnaPortalResponse> portalResponses = new ArrayList<>();
+        EnaPortalResponse enaPortalResponse1 = new EnaPortalResponse();
+        enaPortalResponse1.setParentId("SRX2000905");
+        enaPortalResponse1.setRunId("SRR4000583");
+        enaPortalResponse1.setBytes("1174738707");
+        enaPortalResponse1.setUrl("https://www.ebi.ac.uk/vol1/fastq/SRR400/003/SRR4000583/SRR4000583.fastq.gz");
```
Comment (suranjayathilaka) [inline command-line-downloader/src/test/java/uk/ac/ebi/ena/backend/EnaPortalServiceTest.java:77]: this is not a correct url

File: javafx-gui-downloader/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Note: 
-
-The GUI elements of this project was written using JavaFX. JavaFX is not included by default with OpenJDK and Oracle JDK 11 and newer. 
+The project contains two submodules :
+1. command-line-downloader : User runnable tool for getting data files idempotently and resiliently from ENA. It allows users to download for accessions read files fastq ,read files submitted ,analysis files submitted and analysis files generated.
```
Comment (suranjayathilaka) [inline javafx-gui-downloader/README.md:3]: drop the 2nd sentence

File: command-line-downloader/src/test/java/uk/ac/ebi/ena/backend/AccessionDetailsServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+
+    private List<EnaPortalResponse> getPortalResponses() {
+        List<EnaPortalResponse> portalResponses = new ArrayList<>();
+        EnaPortalResponse enaPortalResponse1 = new EnaPortalResponse();
+        enaPortalResponse1.setParentId("SRX2000905");
+        enaPortalResponse1.setRunId("SRR4000583");
+        enaPortalResponse1.setBytes("1174738707");
+        enaPortalResponse1.setUrl("https://www.ebi.ac.uk/vol1/fastq/SRR400/003/SRR4000583/SRR4000583.fastq.gz");
```
Comment (suranjayathilaka) [inline command-line-downloader/src/test/java/uk/ac/ebi/ena/backend/AccessionDetailsServiceTest.java:87]: these?

**Comment by swatisuman1990** [comment]
Updated readme at the top level with command-line-downloader : https://github.com/enasequence/ena-ftp-downloader/pull/12

**Pull request merged**